src/core/Akka.Streams.Tests/Dsl/FramingSpec.cs

```diff
@@ -436,13 +439,13 @@ public void Length_field_based_framing_must_fail_the_stage_on_negative_length_fi
                 .Via(Flow.Create<ByteString>().Via(Framing.LengthField(4, 1000)))
                 .RunWith(Sink.Seq<ByteString>(), Materializer);
 
-            result.Invoking(t => t.AwaitResult())
+            result.Invoking(async t => await t.ShouldCompleteWithin(3.Seconds()))
```

Comment:
This should be:
```
await Awaiting(async () => await result)
    .Should().ThrowAsync<Framing.FramingException>()
    .WithMessage("Decoded frame header reported negative size -4")
    .ShouldCompleteWithin(3.Seconds());
```

Reply:
To use `Awaiting` you need to add `using static FluentAssertions.FluentActions;`